packages/mongodb-dialects/mongosh/src/test/kotlin/com/mongodb/jbplugin/dialects/mongosh/query/FindTest.kt

```diff
@@ -257,7 +257,7 @@ class FindTest {
             var collection = ""
             var database = ""
 
-            db.getSiblingDB(database).getCollection(collection).find({}).sort({"a": 1, })
+            db.getSiblingDB(database).getCollection(collection).find({}).sort({"a": 1, }).limit(50).toArray()
```

Comment:
Now we force a limit of 50 by default if there is a sort because we were running queries returning thousands of documents in the background, which doesn't make much sense.

Reply:
Also, the toArray() is a cause of the following ticket MONGOSH-2002.